tests/test_io.py

```diff
@@ -40,33 +40,3 @@ def test_get_s3_keys_for_dem():
         'Prefix/GRE240m_sp.tif',
     ]
     assert sorted(io._get_s3_keys_for_dem('Prefix', 'GRE240m')) == sorted(expected)
```

Comment:
attempt a test for `io.subset_jpl_tifs()`?

Reply:
I think for now I'd rather not -- JPL controls the files and has freedom to change them at will. The shapefile is the main thing we need to ensure looks the way we expect, but any of these TIF values we subset, they are likely to be fragile and changes are no necessarily indicative of a failure. I'm not sure it's work checking effectively that just an output file exists from the gdal.Warp.